src/components/sbb-journey-summary/sbb-journey-summary.stories.js

```diff
@@ -0,0 +1,74 @@
+import { h } from 'jsx-dom';
+
+const Template = (args) => (
+  <sbb-journey-summary summaryConfig={args}>
+    <div
+      style={{
+        display: 'flex',
+        paddingTop: '24px',
```

Comment:
I would give our slot the spacing.

you should easily select the slot via ::slotted in our scss.